docs/configuration/scripts.mdx

```diff
@@ -77,7 +77,7 @@ scripts:
       concurrency: 1
 ```
 
-See the [`select-package`](#scriptsselect-package) option for filtering the packages to execute
+See the [`select-package`](scripts#scriptsselect-package) option for filtering the packages to execute
```

Comment:
The URL needs to be `/configuration/scripts#scriptsselect-package`.

Reply:
I'm so stupid.